tools/tune/eq/eq_load_dsp_ini.m

```diff
@@ -0,0 +1,105 @@
+function eq_load_dsp_ini(h, evnt)
+
+udata = get(h, "userdata");
+EQ = udata.EQ;
+preamble_offset = 4;
+config_size = 4;
+n_channels = 2;
+eq = eq_defaults();
+
+[name, path, idx] = uigetfile("*.ini");
+if name == 0
+  % cancelled
+  return;
+endif
+
+% load file
+f = fopen(strcat(path, name));
+
+% locate eq2 section
+do
+  l = fgets(f);
+until strcmp(l, "[eq2]\n") || l == -1
```

Comment:
It the section there always or should there be an error shown at here and in next do loop if file ended?

Reply:
That is a good point, I will follow up with the audio team to see if it always present.

Reply:
added error dialog is search fails